app/gui2/src/components/widgets/DropdownWidget.vue

```diff
@@ -63,21 +63,22 @@ export interface DropdownEntry {
 <template>
   <div class="DropdownWidget" :style="{ '--dropdown-bg': color }">
     <ul class="list scrollable" @wheel.stop>
-      <template v-for="entry in sortedValues" :key="entry.value">
-        <li v-if="entry.selected">
-          <div class="item selected" @click.stop="emit('clickEntry', entry, $event.altKey)">
-            <span v-text="entry.value"></span>
-          </div>
-        </li>
-        <li v-else class="item" @click.stop="emit('clickEntry', entry, $event.altKey)">
-          <span v-text="entry.value"></span>
-        </li>
-      </template>
+      <li
+        v-for="entry in sortedValues"
+        :key="entry.value"
+        :class="{ selected: entry.selected }"
+        class="item clickable"
+        @click.stop="emit('clickEntry', entry, $event.altKey)"
+      >
+        <div v-if="entry.selected"><span v-text="entry.value"></span></div>
+        <span v-else v-text="entry.value"></span>
```

Comment:
Do we need this <div> when item is selected? The div does not seem to have any styling attached (in contrast to previous implementation).